### PR TITLE
Add Channel Banner Feature Flag

### PR DESCRIPTION
**Pull request by Lulalaby**

### CODE REVIEW

**Review by FrostByte266 — APPROVED** [review]

**Comment by FrostByte266** [comment]
> Shouldn't this be added as an attribute too?

Actually, I completely missed this. It does need to be added under the `GuildFeatures` type in `discord/types/guild.py`

**Review by FrostByte266 — CHANGES_REQUESTED** [review]
`CHANNEL_BANNER` needs to be added to the `GuildFeature` type in `discord/types/guild.py`

File: discord/guild.py [hunk 1]
```diff
@@ -202,6 +202,7 @@ class Guild(Hashable):
         - ``ANIMATED_BANNER``: Guild can upload an animated banner.
         - ``ANIMATED_ICON``: Guild can upload an animated icon.
         - ``BANNER``: Guild can upload and use a banner. (i.e. :attr:`.banner`)
+        - ``CHANNEL_BANNER``: Guild can upload and use a channel banners.
```
Comment (Dorukyum) [inline discord/guild.py:205]:
```suggestion
        - ``CHANNEL_BANNERS``: Guild can upload and use channel banners.
```
I suppose this is the correct form.

**Review by Dorukyum — CHANGES_REQUESTED** [review]

File: discord/guild.py (hunk 1 above)
Comment (Lulalaby) [reply discord/guild.py:205]: You can't just change the name 😅

**Comment by Lulalaby** [comment]
> 
> 
> `CHANNEL_BANNER` needs to be added to the `GuildFeature` type in `discord/types/guild.py`

That's not in the scope of the pr.
Instead it should be added in #423

File: discord/guild.py (hunk 1 above)
Comment (Dorukyum) [reply discord/guild.py:205]: My bad, I thought that would be the correct form. Just saw https://github.com/discord/discord-api-docs/pull/4068.

**Review by Dorukyum — APPROVED** [review]

**Review by FrostByte266 — APPROVED** [review]

**Pull request merged**